src/main/scala/com/kifi/franz/SQSQueue.scala

```diff
@@ -81,6 +72,27 @@ trait SQSQueue[T]{
     p.future
   }
 
+   def attributes(attributeNames:List[String]):Future[Map[String,String]]={
```

Comment:
I would prefer a `Seq[String]` as input, to make it slightly more generically usable.